packages/react-native/README.md

```diff
@@ -1,25 +1,83 @@
-# Babel-based transpiling environment for React Native components
+# React Native Compiler
 
-Bit build enviroment for transpiling React Native components using Bit.
+A ReactNative component compiler for [Bit](https://github.com/teambit/bit).
+This compiler follows our best practices guide for compiling shared components.
 
 ## How to use?
 
-Import the environment.
+In order to run this extension your must have a [bit workspace](https://docs.bit.dev/docs/concepts#bit-workspace) with at least one component defined, for more information on how to build please read the [docs](https://docs.bit.dev/docs/building-components) section on the bit website. TL;DR version:
```

Comment:
please add a link to the typescript compiler

Reply:
Why adding a link to the TS compiler?